app/pages/reset-password/reset-password.jsx

```diff
@@ -64,18 +64,35 @@ class ResetPasswordPage extends React.Component {
     const token = this.props.location.query.reset_password_token;
     const password = event.target[0].value;
     const confirmation = event.target[1].value;
+    const passwordMatch = password === confirmation
```

Comment:
I wasn't sure the best way to test form submission behaviour via enzyme. Happy to add testing in here if folks feel strongly about it.